src/js/components/time.jsx

```diff
@@ -10,36 +10,36 @@ const helpers = require('../../../convenience-functions/parseData.js');
 class Time extends React.Component {
```

Comment:
Since you aren't handling state here anymore, should you consider turning this into a functional component instead of a class component?